File: app/src/main/java/org/imozerov/streetartview/network/endpoints/ArtWorksEndpoint.java
package org.imozerov.streetartview.network.endpoints;

import org.imozerov.streetartview.network.model.Artwork;

import retrofit.Call;
import retrofit.http.GET;

import java.util.List;

/**
 * Created by imozerov on 16.02.16.
 */
public interface ArtWorksEndpoint {
    @GET("artworks/")
    Call<List<Artwork>> list();
}

File: app/src/main/java/org/imozerov/streetartview/network/endpoints/ArtWorksEndpoint.kt
package org.imozerov.streetartview.network.endpoints

import org.imozerov.streetartview.network.model.Artwork

import retrofit.Call
import retrofit.http.GET
import rx.Observable

interface ArtWorksEndpoint {
    @GET("artworks/")
    fun list(): Observable<List<Artwork>>
}
